pkg/lang/ir/types.go

```diff
@@ -55,6 +55,7 @@ type Graph struct {
 	Mount           []MountInfo
 	Entrypoint      []string
 	RuntimeCommands map[string]string
+	RuntimeDaemon   []string
```

Comment:
Should it be `[][]string`?

Users may define more than one daemon process.

Reply:
```docker
# 1
[["date", "-u"], ["sleep", "infinity"]]
# 2
["date -u", "sleep infinity"]
```
Which one do you prefer?

Reply:
1 is better for me.

Reply:
Fixed. PTAL.